### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by MinChiHuang**
https://carbon-tutorial-vue-minchihuang.mybluemix.net/

### CODE REVIEW

**Comment by lee-chase** [comment]
:cold_sweat: something doesn't look right. I don't seem to be able to access your ibm cloud deployment.

**Comment by MinChiHuang** [comment]
The application is running now. 
https://carbon-tutorial-vue-minchihuang.mybluemix.net/ 

It was stopped by IBM Cloud at Sep 7, 2019 6:15:15 PM.

File: src/styles/_carbon.scss [hunk 1]
```diff
@@ -1,6 +1,20 @@
 $feature-flags: (
-  grid-columns-16: true
+  grid-columns-16: true,
 );
 
-@import 'carbon-components/scss/globals/scss/styles';
-@import './carbon-fixes';
```
Comment (kellychurchill) [inline src/styles/_carbon.scss]: This is needed. Without it your padding is off.

Comment (MinChiHuang) [reply src/styles/_carbon.scss]: Thanks!

I added the missing styles and the site looks correct.

**Pull request closed without merging**